### PR TITLE
solution

### PR DESCRIPTION
**Pull request by olegpaunov**

### CODE REVIEW

File: src/calculateRentalCost.js [hunk 1]
```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const result = days * 40;
```
Comment (vserhiichuk) [inline src/calculateRentalCost.js:31]: 1. [Checklist] Make names descriptive, so they explain the purpose of variables and functions.
and for the next lines

File: src/calculateRentalCost.js [hunk 2]
```diff
@@ -28,7 +28,15 @@
...
+
+  if (days < 3) {
```
Comment (vserhiichuk) [inline src/calculateRentalCost.js:33]: 2. [Checklist] Don't use literals in expressions (HARDCODE). Instead use constants explaining this values
and for the next lines

File: src/calculateRentalCost.js [hunk 3]
```diff
@@ -28,7 +28,15 @@
...
+    return result;
+  } else if (days >= 3 && days < 7) {
```
Comment (vserhiichuk) [inline src/calculateRentalCost.js:35]: 4. [Checklist] Prefer `if` with return over `if else` to simplify later conditions.

File: src/calculateRentalCost.js [hunk 4]
```diff
@@ -28,7 +28,15 @@
...
+    return result - 20;
```
Comment (vserhiichuk) [inline src/calculateRentalCost.js:36]: 5. [Checklist] DON'T add `else` after `if` with return - the code after it won't be executed anyway.

**Review by vserhiichuk — CHANGES_REQUESTED** [review]
Looks great..)
Few more things to fix in accordance to the checklist
And please look at the checklist again

File: src/calculateRentalCost.js (hunk 1 above)
Comment (olegpaunov) [reply src/calculateRentalCost.js:31]: are u seriosly ? -_-

Comment (vserhiichuk) [reply src/calculateRentalCost.js:31]: I think it would be better to describe what you return as a final result (totalCost)

Comment (alexander-ignatow) [reply src/calculateRentalCost.js]: @vserhiichuk @olegpaunov 

Try to be more clear in the comments. Describe the error more precisely, and when you don't agree with the comment please provide arguments why you should leave it as it is

File: src/calculateRentalCost.js [hunk 5]
```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
```
Comment (alexander-ignatow) [inline src/calculateRentalCost.js:35]: Declare the constants outside of the function body if possible
```
 const PRICE_PER_DAY = 40;
  const SHORT_TERM_DISCOUNT = 20;
  const SHORT_TERM = 3;
  const LONG_TERM_DISCOUNT = 50;
  const LONG_TERM = 7;

function calculateRentalCost(days) {
...
 ```

**Review by alexander-ignatow — COMMENTED** [review]
LGTM!

Good job fixing comments on the first iteration.

**Pull request closed without merging**